client/src/lib/styles.css

```diff
@@ -0,0 +1,3 @@
+.orange-gradient {
```

Comment:
This class will not be scoped, why not just keep it in app.css then ? You'll be rarely using a custom css class anyway since you are using tailwind.